packages/nimble-components/src/menu-button/tests/menu-button-matrix.stories.ts

```diff
@@ -35,39 +40,83 @@ const partVisibilityStates = [
 ] as const;
 type PartVisibilityState = (typeof partVisibilityStates)[number];
 
+const openStates = [
+    ['', false],
+    ['Open', true]
+] as const;
+type OpenState = (typeof openStates)[number];
+
 const appearanceStates = Object.entries(ButtonAppearance).map(
     ([key, value]) => [pascalCase(key), value]
 );
 type AppearanceState = (typeof appearanceStates)[number];
 
 // prettier-ignore
 const component = (
+    [interactionName, interaction]: InteractionState,
     [iconVisible, labelVisible, endIconVisible]: PartVisibilityState,
+    [openName, open]: OpenState,
     [disabledName, disabled]: DisabledState,
     [appearanceName, appearance]: AppearanceState
 ): ViewTemplate => html`
     <${menuButtonTag}
+        class="${() => interaction}"
         appearance="${() => appearance}"
+        ?open="${() => open}"
         ?disabled=${() => disabled}
         ?content-hidden=${() => !labelVisible}
         style="margin-right: 8px; margin-bottom: 8px;">
             ${when(() => iconVisible, html`<${iconKeyTag} slot="start"></${iconKeyTag}>`)}
-            ${() => `${appearanceName!} Menu Button ${disabledName}`}
+            ${() => `${interactionName} ${openName} ${appearanceName!} Menu Button ${disabledName}`}
             ${when(() => endIconVisible, html`<${iconArrowExpanderDownTag} slot="end"></${iconArrowExpanderDownTag}>`)}
-
-        <${menuTag} slot="menu">
-            <${menuItemTag}>Item 1</${menuItemTag}>
-            <${menuItemTag}>Item 2</${menuItemTag}>
-        </${menuTag}>
     </${menuButtonTag}>
 `;
 
 export const menuButtonThemeMatrix: StoryFn = createMatrixThemeStory(
-    createMatrix(component, [
-        partVisibilityStates,
-        disabledStates,
-        appearanceStates
-    ])
+    createMatrix(
+        component,
+        [
+            nonInteractionStates,
+            partVisibilityStates,
+            openStates,
+            disabledStates,
+            appearanceStates
+        ],
+        // Disabled and open is not a valid state
+        (
+            _interactionState: InteractionState,
+            _partVisibilityState: PartVisibilityState,
+            openState: OpenState,
+            disabledState: DisabledState
+        ) => {
+            return disabledState[0] !== 'Disabled' || openState[0] !== 'Open';
+        }
+    )
+);
+
+export const menuButtonInteractionsThemeMatrix: StoryFn = createMatrixThemeStory(
+    createMatrix(
+        component,
+        [
+            interactionStates,
+            [[false, true, false]],
+            openStates,
+            disabledStates,
+            appearanceStates
+        ],
+        // Only interaction relevant to disabled controls is hover
+        (
+            interactionState: InteractionState,
+            _partVisibilityState: PartVisibilityState,
+            _openState: OpenState,
+            disabledState: DisabledState
+        ) => {
+            return disabledInteractionsFilter(
```

Comment:
Could this use the `disabledInteractionsFilter` directly if the order of the states were changed?

Reply:
Yes, I'll reorder them.